packages/calcite-components/src/components/dialog/dialog.tsx

```diff
@@ -347,6 +347,10 @@ export class Dialog extends LitElement implements OpenCloseComponent {
     if (changes.has("opened") && (this.hasUpdated || this.opened !== false)) {
       this.handleOpenedChange(this.opened);
     }
+
+    if (changes.has("embedded") && (this.hasUpdated || this.embedded !== false)) {
+      this.updateFocusTrapElements();
```

Comment:
@driskull not sure if I need these update checks here...

Reply:
@macandcheese you only need to update focus trap elements if new elements are slotted within the component. So this shouldn't be necessary.